src/Npgsql/PostgresErrorCodes.cs

```diff
@@ -0,0 +1,382 @@
+﻿namespace Npgsql
+{
+    /// <summary>
+    /// Provides constants for PostgreSQL error codes.
+    /// </summary>
+    /// <remarks>
+    /// See http://www.postgresql.org/docs/current/static/errcodes-appendix.html
+    /// </remarks>
+    public static class PostgresErrorCodes
+    {
+        #pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+
+        #region Class 00 - Successful Completion
+        public const string SuccessfulCompletion = "00000";
```

Comment:
Appreciate you adding these, but could you keep one line of whitespace between the region blocks and the neighboring code? Makes for an easier read.